src/bindings/channel_header.ts

```diff
@@ -10,13 +10,8 @@ import {BindingOptions} from './index';
 // getChannelHeaderBindings returns the users command bindings
 export const getChannelHeaderBindings = (options: BindingOptions): AppBinding => {
     const bindings: AppBinding[] = [];
-    if (options.isSystemAdmin) {
-        // only show configuration option if admin has not configured the plugin
-        if (!options.isConfigured) {
-            bindings.push(channelHeaderConfig(options.mattermostSiteUrl));
-        } else if (options.isConnected) {
-            bindings.push(channelHeaderSubscribe(options.mattermostSiteUrl));
-        }
+    if (options.isSystemAdmin && options.isConnected) {
+        bindings.push(channelHeaderSubscribe(options.mattermostSiteUrl));
```

Comment:
I asked this in another PR, but is it possible to have a different indicator of "this user should be able to edit subscriptions"? Like a zendesk admin or something